.env.travis

```diff
@@ -120,10 +120,6 @@ LARAVEL_BLOCKER_BLADE_PLACEMENT_JS='footer_scripts'
 LARAVEL_BLOCKER_USE_TYPES_SEED_PUBLISHED=true
 LARAVEL_BLOCKER_USE_ITEMS_SEED_PUBLISHED=true
 
-# Roles Default Models
-ROLES_DEFAULT_ROLE_MODEL=jeremykenedy\LaravelRoles\Models\Role
-ROLES_DEFAULT_PERMISSION_MODEL=jeremykenedy\LaravelRoles\Models\Permission
```

Comment:
To confirm, this is because I changed the models to extend the packages models... correct?

Reply:
Yes !